src/renderer/ipc.ts

```diff
@@ -76,15 +76,15 @@ function callDcMethod(
   }
 
   const onSuccess = (_ev: any, returnValue: any) => {
-    log.debug(
+    /*log.debug(
       `EVENT_DD_DISPATCH_RETURN_${identifier}_${methodName}`,
       'Got back return: [truncated]'
-    )
-    /*log.debug(
+    )*/
+    log.debug(
       `EVENT_DD_DISPATCH_RETURN_${identifier}_${methodName}`,
       'Got back return: [',
       returnValue
-    )*/
+    )
```

Comment:
Uncomment before merge again

Reply:
Or let's leave it in for debugging?

Reply:
you can put it behind an if statement if its a method that you wanna debug, but I don't see the point into logging everything. especially as this makes the logs contain more sensitive information.